.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+-   repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+    -   id: black
+        args:
+        - --line-length
+        - "100"
+-   repo: https://github.com/timothycrosley/isort
```

Comment:
I have always been using `https://github.com/pre-commit/mirrors-isort` - how does it compare to this one?

Reply:
No idea - got this from https://pre-commit.com/hooks.html.  Which is more standard?

Reply:
yikes idk.  Lets keep this one since its recommended from there